internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -50,6 +50,26 @@ username: %s
 password: %s
 `
 
+const clusterDetails = `
+[Cluster Details]
+`
+
+const databaseUserDetails = `
+[Database User Details]
```

Comment:
[q] is the copy coming from design here?

Reply:
Are you asking why I added [Database User Details], [Cluster Details], etc? if so, because of your comment about grouping the questions -> maybe I misunderstood your point

Reply:
question not on the why but the actual copy/text, is the copy coming from Design or is only yours? if only I have opinions to give the user agency, something like
> [Set up your database access user details]

Reply:
oh, thanks for the clarification. No, it is coming from me. Happy to change it

Reply:
so yes, let's make this more about actions
> [Set up your Atlas Cluster]
> [Set up you database access details]
> [Connect to your new cluster]